ArtOfIllusion/src/artofillusion/MoveViewTool.java

```diff
@@ -126,7 +132,8 @@ private void dragMoveTravel(WidgetMouseEvent e, ViewerCanvas view)
 		dx = dragPoint.x-clickPoint.x;
 		dy = dragPoint.y-clickPoint.y;
 
-		if (controlDown) // forward move!
+
+		if (mouseButtonThree(e) && e.isControlDown()) // forward move!
```

Comment:
```suggestion
		if (controlDown) // forward move!
```

The forward move should also be accessible by mouse button 1, when the tool is selected by the tool icon. On the other hand you can not be here if you use MB2, so the button detection is obsolete in that sense.

Reply:
True on both of these.